src/modules/ruby.rs

```diff
@@ -38,7 +38,10 @@ fn format_ruby_version(ruby_version: &str) -> Option<String> {
         .split_whitespace()
         // return "2.6.0p0"
         .nth(1)?
-        .get(0..5)?;
+        // split into ["2.6.0", "0"]
+        .split('p')
+        // return "2.6.0"
+        .nth(0)?;
```

Comment:
It's not obvious from the CI output but I think this line should be
```suggestion
        .next()?;
```

See https://rust-lang.github.io/rust-clippy/master/index.html#iter_nth_zero

Reply:
Sounds good; force pushed an update